docs/install.sh

```diff
@@ -26,8 +25,7 @@ fi
 
 echo "INFO: verifying cpu architecture compatibility"
 
-ARCH=$(uname -m)
-ARCH=$(echo "${ARCH,,}") # convert to lower case
+ARCH=$(uname -m | tr '[:upper:]' '[:lower:]')
```

Comment:
Were these changed because they were causing some problem? `tr` is a GNU coreutil, is it present on a Mac system (I don't use Mac so I know very little about them except that they're originally FreeBSD kernel based)?

Reply:
It didn't work for me on Mac - probably because it has version 3.2 - (while `tr` does) nor on a VM with BusyBox (since it uses `ash`).